src/job_manager/views/job.py

```diff
@@ -0,0 +1,257 @@
+        ],
+        "show_edit_actions": False,
+    },
+    "dependencies": {
+        "model_name": "dependency",
+        "related_name": "dependencies",
+        "headers": [
+            "ID",
+            "Dependency Name",
+            "Expected Close",
+            "Actual Close",
+            "Type",
+            "Status",
+        ],
+        "fields": [
+            "id",
+            "name",
+            "expected_close_datetime",
+            "actual_close_datetime",
+            "dependency_type",
+            "dependency_status",
+        ],
+        "show_edit_actions": False,
+    },
+    "history": {
+        "model_name": "history",
+        "related_name": "history",
+        "headers": [
+            "ID",
+            "Name",
+            "User",
+            "Customer",
+            "Due Date",
+            "Description",
+            "Notes",
+            "History Date",
+        ],
+        "fields": [
+            "history_id",
+            "name",
+            "history_user",
+            "customer",
+            "due_date",
+            "description",
+            "notes",
+            "history_date",
+        ],
+        "show_edit_actions": False,
+    },
+}
+
+
+JOB_TABLE_VIEW = CustomTableView(
+    model=Job,
+    model_name="job",
+    fields=JOB_MODEL_FIELDS,
+    status_choices_class=JobStatusChoices,
+    model_relation_headers=JOB_MODEL_RELATION_HEADERS,
+    model_relation_fields=JOB_MODEL_RELATION_FIELDS,
+    status_filter_field=JOB_STATUS_FILTER_FIELD,
+    search_fields_list=JOB_SEARCH_FIELDS,
+    tailwind_classes=JOB_TAILWIND_CLASSES,
+)
+
+
+JOB_VIEWS = CRUDView(
+    model=Job,
+    model_type=JobType,
+    model_name="jobs",
+    model_service=JobService,
+    model_form=JobForm,
+    model_table_view=JOB_TABLE_VIEW,
+)
+
+
+# ------------------------------------------------------------------------------
+# Job Priority manager
+# ------------------------------------------------------------------------------
+
+
+def job_prioritization_view(request):
+    """
+    Job Prioritization View
+    """
+    if request.user.require_password_change:
+        return redirect(reverse("users:change_password"))
+
+    if request.htmx:
+        try:
+            job_data = json.loads(request.POST.get("job_data"))
+
+            with transaction.atomic():
+                for job in job_data:
+                    job_obj = Job.objects.get(id=job["id"])
+                    if "due_date" in job:
+                        job_obj.due_date = job["due_date"]
+                    if "manual_priority" in job:
+                        job_obj.manual_priority = job["manual_priority"]
+                    job_obj.save()
+        except Exception as e:
+            response = HttpResponse(status=400)
+            add_notification_headers(
+                response,
+                str(e),
+                "success",
+            )
+
+    jobs = (
+        Job.objects.exclude(job_status__in=["CM", "CN"])
```

Comment:
Let's use the enum values here, instead of the raw strings ["CM", "CN"].
Something like this: 
```
job_status__in=[status.NOT_PLANNED, status.CANCELLED]
```